buildscripts/azure/azure-windows.yml

```diff
@@ -62,5 +64,8 @@ jobs:
         numba -s
         popd
         python -m numba.tests.test_runtests
-        python runtests.py -m 2 -b -- numba.tests
+        # Now run tests based on the changes identified via git
+        python -m numba.runtests -b -v -g -m 2 -- numba.tests
```

Comment:
Do we want to hardcode the `2` here or check the number of cores on the machine and use that value?